---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Comparative Analysis of Metagenomics and Metataxonomics for the Characterization
  of Vermicompost Microbiomes
subtitle: ''
summary: ''
authors:
- Marcos Pérez-Losada
- Dhatri Badri Narayanan
- Allison R. Kolbe
- Ignacio Ramos-Tapia
- Eduardo Castro-Nallar
- Keith A. Crandall
- Jorge Domínguez
tags: []
categories: []
date: '2022-05-01'
lastmod: 2023-02-27T14:49:56-03:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2023-02-27T17:49:56.159484Z'
publication_types:
- '2'
abstract: The study of microbial communities or microbiotas in animals and environments
  is important because of their impact in a broad range of industrial applications,
  diseases and ecological roles. High throughput sequencing (HTS) is the best strategy
  to characterize microbial composition and function. Microbial profiles can be obtained
  either by shotgun sequencing of genomes, or through amplicon sequencing of target
  genes (e.g., 16S rRNA for bacteria and ITS for fungi). Here, we compared both HTS
  approaches at assessing taxonomic and functional diversity of bacterial and fungal
  communities during vermicomposting of white grape marc. We applied specific HTS
  workflows to the same 12 microcosms, with and without earthworms, sampled at two
  distinct phases of the vermicomposting process occurring at 21 and 63 days. Metataxonomic
  profiles were inferred in DADA2, with bacterial metabolic pathways predicted via
  PICRUSt2. Metagenomic taxonomic profiles were inferred in PathoScope, while bacterial
  functional profiles were inferred in Humann2. Microbial profiles inferred by metagenomics
  and metataxonomics showed similarities and differences in composition, structure,
  and metabolic function at different taxonomic levels. Microbial composition and
  abundance estimated by both HTS approaches agreed reasonably well at the phylum
  level, but larger discrepancies were observed at lower taxonomic ranks. Shotgun
  HTS identified ~1.8 times more bacterial genera than 16S rRNA HTS, while ITS HTS
  identified two times more fungal genera than shotgun HTS. This is mainly a consequence
  of the difference in resolution and reference richness between amplicon and genome
  sequencing approaches and databases, respectively. Our study also revealed great
  differences and even opposite trends in alpha- and beta-diversity between amplicon
  and shotgun HTS. Interestingly, amplicon PICRUSt2-imputed functional repertoires
  overlapped ~50% with shotgun Humann2 profiles. Finally, both approaches indicated
  that although bacteria and fungi are the main drivers of biochemical decomposition,
  earthworms also play a key role in plant vermicomposting. In summary, our study
  highlights the strengths and weaknesses of metagenomics and metataxonomics and provides
  new insights on the vermicomposting of white grape marc. Since both approaches may
  target different biological aspects of the communities, combining them will provide
  a better understanding of the microbiotas under study.
publication: '*Frontiers in Microbiology*'
doi: 10.3389/fmicb.2022.854423
links:
- name: URL
  url: https://www.frontiersin.org/articles/10.3389/fmicb.2022.854423/full
---
